snap-wrappers/bin/set-xdg-data-home

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export XDG_DATA_HOME=$SNAP_USER_DATA/data
+exec "$@"
```

Comment:
This is 100% a hack. That variable is overwritten in another command that gets added to the chain by the `gnome` extension, and we can't make it not do that. But fortunately, that command gets added before any other commands added manually to the command chain, so we can overwrite the overwrite. The weird part is the `exec "$@"`. I couldn't find any documentation about this, but apparently each command in the command chain must call the next one through this `exec`. Otherwise, the final command that opens up the GUI simply won't be called. From what I've seen, other commands in command chains also do this, so I assume that's just how it is.

Reply:
Crazy. OK thank you for clarifying that. I understand also that this was already discussed with the desktop team and they said that the gnome extension was like that by design, so nothing else we can do.